docs/experience/longtasks.md

```diff
@@ -0,0 +1,65 @@
+title: Long Tasks
+description: Gathering information about long tasks during hard and soft page loads
+
+Experience collects information about long tasks that happen during page loads. According to the RAIL Model, tasks that are running for longer than **50ms** and blocking the main thread are considered long and are recorded by Sematext Cloud. 
+
+Long tasks can cause issues such as:
+
+* Clunky animations and scrolling affecting user experience.
+* High input latency.
+* Long delay in events handling.
+* Delayed time when your application becomes interactive.
+
+Because of that, it is crucial to see the long tasks that are running when your application handles your user requests.
+
+## Page Load Overview
+The aggregated data about long tasks are visible is the page loads overview of your Experience App. You can check the number of long tasks and their average duration:
+
+<img
+  class="content-modal-image"
+  alt="Page Loads Overview"
+  src="../../images/experience/longtasks/overview.png"
+  title="Page Loads Overview"
+  width=778
+/>
+
+
+## Page Load Details
+In addition to the page loads overview where the long tasks overview is visible each of your page loads contains information about long tasks that happened during both the hard and soft page loads. Each long task is described by its **name**, **timestamp**, and **duration**.
+
+The **name** of the long task can point to an even of the page load that was executed for a longer period of time. The value of the can be set to one of the following values:
```

Comment:
even => event?
But what would "event of a page load" be?

The value of the can => something's off here

Reply:
Adjusted, thanks.